modules/cache/src/main/java/org/eclipse/xpanse/modules/cache/consts/RegionAzCacheConstant.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.cache.consts;
+
+/**
+ * Region az information cache constant class.
+ */
+public class RegionAzCacheConstant {
```

Comment:
we can call this as 'CacheNames' and it can hold names of all caches which we will have in xpanse